### ISSUE TITLE
Internal error after installing Redmine plugin

### ISSUE THREAD

**Issue by yingsheng658** [bitnami/bitnami-docker-redmine#88]
**Description**
Internal error after installing Redmine plugin as https://github.com/onozaty/redmine-view-customize.
Means that cannot access Redmine anymore after installing plugin.

**Steps to reproduce the issue:**
1. Download redmine-view-customize plug-in.
2. Install plugin.
3. Migrate plugin.
4. Restart server.
5. Access the Redmine system.

**Describe the results you received:**
Internal error when access the Redmine system.

**Describe the results you expected:**
Can access the Redmine system normally.

**Version**
Redmine version is v3.3.0-1.

**Production log**:
```
Started GET "/redmine/" for 127.0.0.1 at 2017-04-28 00:33:57 -0700
Processing by WelcomeController#index as HTML
Current user: anonymous
Rendered welcome/index.html.erb within layouts/base (0.0ms)
Completed 500 Internal Server Error in 16ms (ActiveRecord: 0.0ms)

ActionView::Template::Error (Mysql2::Error: Table 'bitnami_redmine.view_customizes' doesn't exist: SELECT `view_customizes`.* FROM `view_customizes`):
13: <%= stylesheet_link_tag 'rtl', :media => 'all' if l(:direction) == 'rtl' >
14: <= javascript_heads >
15: <= heads_for_theme >
16: <= call_hook :view_layouts_base_html_head >
17: <!-- page specific tags -->
18: <= yield :header_tags -%>
19: </head>
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:17:in `match_customize'
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:5:in `view_layouts_base_html_head'
lib/redmine/hook.rb:61:in `block (2 levels) in call_hook'
lib/redmine/hook.rb:61:in `each'
lib/redmine/hook.rb:61:in `block in call_hook'
lib/redmine/hook.rb:58:in `tap'
lib/redmine/hook.rb:58:in `call_hook'
lib/redmine/hook.rb:96:in `call_hook'
app/views/layouts/base.html.erb:16:in `_app_views_layouts_base_html_erb__886651538_46400232'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'

Started GET "/redmine/" for 127.0.0.1 at 2017-04-28 00:33:57 -0700
Processing by WelcomeController#index as HTML
Current user: anonymous
Rendered welcome/index.html.erb within layouts/base (0.0ms)
Completed 500 Internal Server Error in 16ms (ActiveRecord: 0.0ms)

ActionView::Template::Error (Mysql2::Error: Table 'bitnami_redmine.view_customizes' doesn't exist: SELECT `view_customizes`.* FROM `view_customizes`):
13: <%= stylesheet_link_tag 'rtl', :media => 'all' if l(:direction) == 'rtl' >
14: <= javascript_heads >
15: <= heads_for_theme >
16: <= call_hook :view_layouts_base_html_head >
17: <!-- page specific tags -->
18: <= yield :header_tags -%>
19: </head>
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:17:in `match_customize'
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:5:in `view_layouts_base_html_head'
lib/redmine/hook.rb:61:in `block (2 levels) in call_hook'
lib/redmine/hook.rb:61:in `each'
lib/redmine/hook.rb:61:in `block in call_hook'
lib/redmine/hook.rb:58:in `tap'
lib/redmine/hook.rb:58:in `call_hook'
lib/redmine/hook.rb:96:in `call_hook'
app/views/layouts/base.html.erb:16:in `_app_views_layouts_base_html_erb__342461918_47151996'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'

Started GET "/redmine/" for 127.0.0.1 at 2017-04-28 00:34:29 -0700
Processing by WelcomeController#index as HTML
Current user: anonymous
Rendered welcome/index.html.erb within layouts/base (0.0ms)
Completed 500 Internal Server Error in 16ms (ActiveRecord: 0.0ms)

ActionView::Template::Error (Mysql2::Error: Table 'bitnami_redmine.view_customizes' doesn't exist: SELECT `view_customizes`.* FROM `view_customizes`):
13: <%= stylesheet_link_tag 'rtl', :media => 'all' if l(:direction) == 'rtl' >
14: <= javascript_heads >
15: <= heads_for_theme >
16: <= call_hook :view_layouts_base_html_head >
17: <!-- page specific tags -->
18: <= yield :header_tags -%>
19: </head>
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:17:in `match_customize'
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:5:in `view_layouts_base_html_head'
lib/redmine/hook.rb:61:in `block (2 levels) in call_hook'
lib/redmine/hook.rb:61:in `each'
lib/redmine/hook.rb:61:in `block in call_hook'
lib/redmine/hook.rb:58:in `tap'
lib/redmine/hook.rb:58:in `call_hook'
lib/redmine/hook.rb:96:in `call_hook'
app/views/layouts/base.html.erb:16:in `_app_views_layouts_base_html_erb__886651538_46400232'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'

Started GET "/redmine/" for 127.0.0.1 at 2017-04-28 23:47:31 -0700
Processing by WelcomeController#index as HTML
Current user: anonymous
Rendered welcome/index.html.erb within layouts/base (15.6ms)
Completed 500 Internal Server Error in 858ms (ActiveRecord: 202.8ms)

ActionView::Template::Error (Mysql2::Error: Table 'bitnami_redmine.view_customizes' doesn't exist: SELECT `view_customizes`.* FROM `view_customizes`):
13: <%= stylesheet_link_tag 'rtl', :media => 'all' if l(:direction) == 'rtl' >
14: <= javascript_heads >
15: <= heads_for_theme >
16: <= call_hook :view_layouts_base_html_head >
17: <!-- page specific tags -->
18: <= yield :header_tags -%>
19: </head>
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:17:in `match_customize'
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:5:in `view_layouts_base_html_head'
lib/redmine/hook.rb:61:in `block (2 levels) in call_hook'
lib/redmine/hook.rb:61:in `each'
lib/redmine/hook.rb:61:in `block in call_hook'
lib/redmine/hook.rb:58:in `tap'
lib/redmine/hook.rb:58:in `call_hook'
lib/redmine/hook.rb:96:in `call_hook'
app/views/layouts/base.html.erb:16:in `_app_views_layouts_base_html_erb___283165339_46497684'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'

Started GET "/redmine/" for 127.0.0.1 at 2017-04-28 23:50:33 -0700
Processing by WelcomeController#index as HTML
Current user: anonymous
Rendered welcome/index.html.erb within layouts/base (15.6ms)
Completed 500 Internal Server Error in 468ms (ActiveRecord: 62.4ms)

ActionView::Template::Error (Mysql2::Error: Table 'bitnami_redmine.view_customizes' doesn't exist: SELECT `view_customizes`.* FROM `view_customizes`):
13: <%= stylesheet_link_tag 'rtl', :media => 'all' if l(:direction) == 'rtl' >
14: <= javascript_heads >
15: <= heads_for_theme >
16: <= call_hook :view_layouts_base_html_head >
17: <!-- page specific tags -->
18: <= yield :header_tags -%>
19: </head>
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:17:in `match_customize'
plugins/view_customize/lib/view_customize/view_layouts_base_html_head_hook.rb:5:in `view_layouts_base_html_head'
lib/redmine/hook.rb:61:in `block (2 levels) in call_hook'
lib/redmine/hook.rb:61:in `each'
lib/redmine/hook.rb:61:in `block in call_hook'
lib/redmine/hook.rb:58:in `tap'
lib/redmine/hook.rb:58:in `call_hook'
lib/redmine/hook.rb:96:in `call_hook'
app/views/layouts/base.html.erb:16:in `_app_views_layouts_base_html_erb___363638565_46160580'
lib/redmine/sudo_mode.rb:63:in `sudo_mode'
```

**Comment by javsalgar** [comment]
Hi, 

I was unable to reproduce the issue. I was able to install the plugin without problems. This is what I did:

```
cd /opt/bitnami/redmine/plugins/
git clone https://github.com/onozaty/redmine-view-customize.git view_customize
cd ../
bundle exec rake redmine:plugins:migrate RAILS_ENV=production
```

After that, I executed `docker-compose restart redmine` and everything worked without problems

<img width="736" alt="screen shot 2017-05-08 at 17 04 05" src="https://cloud.githubusercontent.com/assets/1133777/25810993/13b09cf2-3412-11e7-8977-314ed8d74875.png">

Did  you execute the same set of commands in your container?
